### PR TITLE
Added release blurb and fixed grammar in api breaking sentence

### PR DESCRIPTION
**Pull request by Qchristensen**
### Summary of the issue:
Needed a release blurb for 2021.1 and also there was a grammar error in the "API breaking sentence" in the what's new document.

### Description of how this pull request fixes the issue:

With words

### Testing strategy:

Read text, and ran past others when writing.

### Change log entries:

For Developers

### Code Review Checklist:

- [X] Pull Request description is up to date.
- [ ] Unit tests.
- [ ] System (end to end) tests.
- [ ] Manual tests.
- [ ] User Documentation.
- [X] Change log entry.
- [ ] Context sensitive help for GUI changes.

### CODE REVIEW

**Comment by josephsl** [comment]
Hi,

In the release blurb: I think there is still a grammatical error with "add-on API" sentence, and may I suggest splitting that note into a separate line/paragraph?

Thanks.

File: user_docs/en/changes.t2t [hunk 1]
```diff
@@ -4,6 +4,8 @@ What's New in NVDA
 %!includeconf: ../changes.t2tconf
 
 = 2021.1 =
+This release includes optional experimental support for UIA in Excel and
+Chromium browsers. There are fixes for several languages, and for accessing links in Braille. There are updates to Unicode CLDR and maths symbols, eSpeak-NG and LibLouis. Many bug fixes and improvements, including in Office, Visual Studio and several languages. Note: this is a Add-on API compatibility breaking release. Add-ons will need to be re-tested and have their manifest updated. This release also drops support for Adobe Flash.
```
Comment (feerrenrut) [inline user_docs/en/changes.t2t:8]: We normally put sentences on new lines to make life easier for translators. Also this copied the grammar error before the fix.
```suggestion
This release includes optional experimental support for UIA in Excel and Chromium browsers.
There are fixes for several languages, and for accessing links in Braille.
There are updates to Unicode CLDR and maths symbols, eSpeak-NG and LibLouis.
Many bug fixes and improvements, including in Office, Visual Studio and several languages.
Note: this is an Add-on API compatibility breaking release.
Add-ons will need to be re-tested and have their manifest updated.
This release also drops support for Adobe Flash.

```

**Review by feerrenrut — CHANGES_REQUESTED** [review]

File: user_docs/en/changes.t2t (hunk 1 above)
Comment (codeofdusk) [inline user_docs/en/changes.t2t:8]: Use "mathematics" in international contexts as the abbreviation [differs by English variety](https://en.wiktionary.org/wiki/math#Noun_2). Also fix a/an.

```suggestion
Chromium browsers. There are fixes for several languages, and for accessing links in Braille. There are updates to Unicode CLDR and mathematical symbols, eSpeak-NG and LibLouis. Many bug fixes and improvements, including in Office, Visual Studio and several languages. Note: this is an Add-on API compatibility breaking release. Add-ons will need to be re-tested and have their manifest updated. This release also drops support for Adobe Flash.

```

Comment (Qchristensen) [reply user_docs/en/changes.t2t]: Thanks Bill, I've made that change, and another grammatical error someone pointed out via email.

File: user_docs/en/changes.t2t [hunk 2]
```diff
@@ -4,6 +4,14 @@ What's New in NVDA
 %!includeconf: ../changes.t2tconf
 
 = 2021.1 =
+This release includes optional experimental support for UIA in Excel and Chromium browsers.
+There are fixes for several languages, and for accessing links in Braille.
+There are updates to Unicode CLDR and mathematics symbols, eSpeak-NG and LibLouis.
```
Comment (codeofdusk) [inline user_docs/en/changes.t2t:9]:
```suggestion
There are updates to Unicode CLDR and mathematical symbols, eSpeak-NG and LibLouis.

```

**Review by codeofdusk — APPROVED** [review]

**Review by feerrenrut — APPROVED** [review]
Looks good @Qchristensen

**Pull request merged**